graphql.gemspec

```diff
@@ -36,7 +36,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency "racc", "~> 1.4"
   s.add_development_dependency "rake", "~> 12"
   s.add_development_dependency "rubocop", "0.68" # for Ruby 2.2 enforcement
-  s.add_development_dependency "stackprof"
```

Comment:
FWIW, one possibility might be to add it like `gem "stackprof", platforms: [:mri]` if you need it on MRI.
We'll also look into letting stackprof install, although it's rather unlikely stackprof would work at runtime on TruffleRuby without changes in stackprof: https://github.com/oracle/truffleruby/issues/2044

Reply:
Thanks for the suggestion -- I briefly considered a _proper_ fix, but then I remembered that when I used `ruby-prof` for profiling, it wasn't in the gemspec either 🤷 Nobody seemed to tripped up by it (it's only used by a couple rake tasks). But in any case, I'll give this a try if I turn out to need it -- thanks again!